packages/components/src/styles/components/content-nav.less

```
@@ -0,0 +1,77 @@
+    search-border: color-mix(in srgb, var(--tr-border-color-default) 42%, transparent);
+    search-border-focus: color-mix(in srgb, var(--tr-color-primary) 52%, transparent);
+    search-focus-ring: color-mix(in srgb, var(--tr-color-primary) 16%, transparent);
+    search-radius: var(--tr-radius-md);
+    empty-color: var(--tr-text-tertiary);
+    focus-ring: color-mix(in srgb, var(--tr-color-primary) 58%, transparent);
+    highlight-color: var(--tr-color-primary);
+  };
+
+  :root,
+  [data-tr-theme] {
+    each(@vars, {
```

Stylelint (17.7.0)

[error] Unknown word each (CssSyntaxError)

(CssSyntaxError)